### PR TITLE
style(formatting): technical overview of Operate Components docs

### PR DESCRIPTION
**Pull request by christinaausley**
As part of Trello card/OKR for this quarter, I've reviewed all Operate documentation within Components to ensure uniform styling across all pre-existing AND migrated Camunda Cloud documentation. Particular adjustments include incorporating admonitions on note references, tabs, general stylistic updates, descriptions, and potential document restructures (though this was not applicable within current Operate documentation.)

### CODE REVIEW

**Comment by akeller** [comment]
Checking in here - were you able to find a working example of multiple tabs on a single page? Even though it's a little messy to redeclare them each time, are there any issues? I can take time later this week to look at this if so.

**Comment by christinaausley** [comment]
So I inspected [this Docusaurus doc](https://docusaurus.io/docs/next/advanced/swizzling) you found, and I understand how it's split up by an HTML perspective, but I'm trying to configure a clean way to execute this in Markdown. I'm going to play around with it in a local build, but we can touch base in our 1:1 too.
Screen Shot 2022-02-16 at 8 11 57 AM
Screen Shot 2022-02-16 at 8 12 20 AM

File: docs/components/operate/index.md [hunk 1]
```diff
@@ -19,13 +20,16 @@ dark: useBaseUrl('img/operate-introduction_dark.png'),
 
 In addition to providing visibility into active and completed process instances, Operate also makes it possible to carry out key operations such as resolving <!-- FIXME: [incidents](/reference/incidents.html) --> incidents, and updating process instance variables.
 
+[//]:# (What is the link to fix here?)
```
Comment (akeller) [inline docs/components/operate/index.md:23]: @santanaluiz any idea what link should go in here?

File: docs/components/operate/userguide/delete-finished-instances.md [hunk 2]
```diff
@@ -1,46 +1,51 @@
...
+1. Confirm the delete operation by clicking **Delete**.
 
 ![operate-instance-detail-confirm-delete-operation](./img/operate-instance-detail-delete-operation-confirm_light.png)
 
-5. Process instance is deleted
+The process instance is now deleted.
 
 ![operate-instance-deleted-notification](./img/operate-instance-deleted-notification_light.png)
+
+[//]:# (Is there any chance of getting this process instance back? If not, we should probably note a warning.)
```
Comment (akeller) [inline docs/components/operate/userguide/delete-finished-instances.md:51]: @santanaluiz Is this a hard delete, dropped from the database entirely? Could CCSM users retrieve it from a database?

File: docs/components/operate/userguide/operate-feedback-and-questions.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
 id: operate-feedback-and-questions
 title: Giving feedback and asking questions
+description: "Have questions or feedback about Operate? Contact us."
 ---
 
 Have questions or feedback about Operate?
 
 Contact us at [Camunda Cloud Contact](/contact).
 
+[//]:# (Is this still up to date or are there additional methods for contact currently?)
```
Comment (akeller) [inline docs/components/operate/userguide/operate-feedback-and-questions.md:11]: Linking to the contact page makes sense, but we could redirect to the Contact page and delete this.

**Comment by akeller** [comment]
@christinaausley can you follow up or respond to the open comments on this PR please? Thanks!

File: docs/components/operate/index.md (hunk 1 above)
Comment (santanaluiz) [reply docs/components/operate/index.md:23]: I would guess here:
https://docs.camunda.io/docs/components/operate/userguide/resolve-incidents-update-variables/

no?

File: docs/components/operate/userguide/delete-finished-instances.md (hunk 2 above)
Comment (santanaluiz) [reply docs/components/operate/userguide/delete-finished-instances.md:51]: AFAIK, we can't have access to it again. It is a hard delete, mostly for GDPR purposes

**Comment by christinaausley** [comment]
@akeller Resolved and merging!

**Pull request merged**